lib/bulbs-elements/store/store.js

```diff
@@ -42,13 +65,15 @@ function dispatchFunction (store, action, payload) {
     console.time(`dispatch ${action.type}`);
   }
 
-  let nextState = cloneObject(store.state);
-  let actionState = nextState[action.fieldName];
-  let actionNextState = cloneObject(actionState);
-  actionNextState = action(actionNextState, payload, store);
-  nextState[action.fieldName] = actionNextState;
+  let priorState = store.state;
+  let nextState = Object.assign({}, priorState);
```

Comment:
We do a shallow copy of the store's full state with `Object.assign`.

This way, we can do a strict equals check on `store.state === store.state`. Good for react to know that a component needs to re-render.